HACKING.md

```diff
@@ -0,0 +1,70 @@
+# Hacking on Origin Aggregated Logging
+
+## Building on OpenShift
+Choose the project you want to hold your logging infrastructure. It can be
+any project.  The project/namespace ```logging``` is the default.
+
+Instantiate the [dev-builds template](hack/templates/dev-builds.yaml)
+to define BuildConfigs for all images and ImageStreams to hold their
+output. You can do this before or after deployment, but before is
```

Comment:
We should just say they should do this prior to deployment, especially if they are using the deployer  from a dev-build IS.